app/poros/garden_health.rb

```diff
@@ -0,0 +1,9 @@
+class GardenHealth
+  attr_reader :id, :reading_type, :reading, :created_at
+  def initialize(data)
+    @id = data[:attributes][:id]
+    @reading_type = data[:attributes][:reading_type]
+    @reading = data[:attributes][:reading]
+    @created_at = data[:attributes][:created_at]
+  end
+end
```

Comment:
I'm glad there's no conflicts here. Maybe I should have made a pr sooner so so you didn't have to rewrite this code